generators/languages/templates/src/main/webapp/i18n/ua/_global.json

```diff
@@ -0,0 +1,158 @@
+                "register": "У вас немає облікового запису? <a class=\"alert-link\" href=\"\" ng-click=\"vm.register()\">Створити новий обліковий запис</a>"
+                <%_ } else { _%>
+                "authenticated": {
+                    "prefix": "Якщо ви хочете",
+                    "link": "авторизуватися",
+                    "suffix": ", ви можете спробувати облікові записи за замовчуванням:<br/>- Адмiністратор (логін=\"admin\" і пароль=\"admin\") <br/>- Користувач (логін=\"user\" і пароль=\"user\")."
+                },
+                "register": {
+                    "noaccount": "У вас немає облікового запису?",
+                    "link": "Створити новий обліковий запис"
+                }
+                <%_ } _%>
+            },
+            "error": {
+                "dontmatch": "Пароль і його підтвердження не збігаються!"
+            },
+            "validate": {
+                "newpassword": {
+                    "required": "Пароль необхідно вказати.",
+                    "minlength": "Довжина пароля повинна бути хоча б 5 символів",
+                    "maxlength": "Пароль не може бути довшим ніж 50 cимволів",
+                    "strength": "Складність пароля:"
+                },
+                "confirmpassword": {
+                    "required": "Підтвердження пароля необхідно вказати.",
+                    "minlength": "Довжина підтвердження пароля повинна бути хоча б 5 символів",
+                    "maxlength": "Підтвердження пароля не може бути довшим ніж 50 cимволів"
+                },
+                "email": {
+                    "required": "Електронну пошту необхідно вказати",
+                    "invalid": "Eлектронна пошта невірна",
+                    "minlength": "Довжина електронної пошти повинна бути хоча б 5 символів",
+                    "maxlength": "Довжина електронної пошти не може бути більшою, ніж 50 cимволів"
+                }
+            }
+        },
+        "field": {
+            "id" : "ID"
+        },
+        "ribbon": {
+            "dev":"Розробка"
+        },
+        "item-count": "Показано {{first}} - {{second}} з {{total}} пунктів."
+    },
+    "entity": {
+        "action": {
+            "addblob": "Додати двійковий об'єкт",
+            "addimage": "Додати зображення",
+            "back": "Назад",
+            "cancel": "Відміна",
+            "delete": "Видалити",
+            "edit": "Змінити",
+            "open": "Відкрити",
+            "save": "Зберегти",
+            "view": "Перегляд"
+        },
+        "detail": {
+            "field": "Поле",
+            "value": "Значення"
+        },
+        "delete": {
+            "title": "Підтвердіть операцію видалення"
+        },
+        "validation": {
+            "required": "Це поле є обов'язковим для заповнення.",
+            "minlength": "Це поле є обов'язковим, принаймні {{min}} символів.",
```

Comment:
Not agree. https://ukrainskamova.com/search/?q=%D0%9F%D1%80%D0%B8%D0%BD%D0%B0%D0%B9%D0%BC%D0%BD%D1%96&t=0